.github/CODEOWNERS

```diff
@@ -69,6 +69,10 @@ components/weekly_storage/ @iefremov
 # Perf predictor
 components/brave_perf_predictor/ @iefremov
 
+# Permissions
+browser/permissions/ @akhoroshilov
```

Comment:
we really need to avoid having entire components blocked on one person, can you restrict this to just critical parts of the code and/or add more people?